spec/models/pxe_menu_ipxe_spec.rb

```diff
@@ -87,8 +87,10 @@
   end
 
   context "#synchronize_images" do
+    let(:auth) { FactoryBot.create(:authentication) }
+
     before do
-      @pxe_server = FactoryBot.create(:pxe_server)
+      @pxe_server = FactoryBot.create(:pxe_server, :authentications => [auth])
```

Comment:
Instead of forcing an array context, it's better to do this:

```
let(:auth) { FactoryBot.create_list(:authentication, 1) }

@pxe_server = FactoryBot.create(:pxe_server, :authentications => auth)
```
That way you have an actual AR relation instead of an array.